bundle/manifests/kepler.system.sustainable.computing.io_keplerinternals.yaml

```diff
@@ -0,0 +1,239 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.12.1
+  creationTimestamp: null
+  name: keplerinternals.kepler.system.sustainable.computing.io
+spec:
+  group: kepler.system.sustainable.computing.io
+  names:
+    kind: KeplerInternal
+    listKind: KeplerInternalList
+    plural: keplerinternals
+    singular: keplerinternal
+  scope: Cluster
+  versions:
+  - additionalPrinterColumns:
+    - jsonPath: .spec.exporter.deployment.port
+      name: Port
+      type: integer
+    - jsonPath: .status.desiredNumberScheduled
+      name: Desired
+      type: integer
+    - jsonPath: .status.currentNumberScheduled
+      name: Current
+      type: integer
+    - jsonPath: .status.numberReady
+      name: Ready
+      type: integer
+    - jsonPath: .status.updatedNumberScheduled
+      name: Up-to-date
+      type: integer
+    - jsonPath: .status.numberAvailable
+      name: Available
+      type: integer
+    - jsonPath: .metadata.creationTimestamp
+      name: Age
+      type: date
+    - jsonPath: .spec.exporter.deployment.image
+      name: Image
+      type: string
+    - jsonPath: .spec.exporter.deployment.nodeSelector
+      name: Node-Selector
+      priority: 10
+      type: string
+    - jsonPath: .spec.exporter.deployment.tolerations
+      name: Tolerations
+      priority: 10
+      type: string
+    name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        description: KeplerInternal is the Schema for the keplers internal API
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: KeplerInternalSpec defines the desired state of Kepler
+            properties:
+              exporter:
+                properties:
+                  deployment:
+                    properties:
+                      image:
```

Comment:
Shouldn't we add a default image `quay.io/sustainable_computing_io/kepler:latest` for `keplerinternals` ?

Reply:
Yeah, we could do that but it does we may be running *any* image. My preference would be to let user decide.